src/Illuminate/View/Compilers/Concerns/CompilesConditionals.php

```diff
@@ -98,4 +105,52 @@ protected function compileEndIsset()
     {
         return '<?php endif; ?>';
     }
+
+    /**
+     * Compile the switch statements into valid PHP.
+     *
+     * @param  string  $variable
+     * @return string
+     */
+    protected function compileSwitch($variable)
+    {
+        return "<?php switch ({$variable}): ";
+    }
+
+    /**
+     * Compile the case statements into valid PHP.
+     *
+     * @param  string  $value
+     * @return string
+     */
+    protected function compileCase($value)
+    {
+        if ($this->firstCaseInSwitch) {
+            $this->firstCaseInSwitch = false;
```

Comment:
This'll break with multiple switch statements in a file, right? It'll print `<?php switch($x): <?php case...`.

Can the `firstCaseInSwitch` be removed and have `compileSwitch` just close the `?>`?